src/main/java/DialogBox.java

```diff
@@ -0,0 +1,105 @@
+//import javafx.collections.FXCollections;
+//import javafx.collections.ObservableList;
+//import javafx.scene.Node;
+//import javafx.scene.control.Label;
+//import javafx.scene.image.ImageView;
+//import javafx.geometry.Pos;
+//import javafx.scene.layout.HBox;
+//
+//public class DialogBox extends HBox{
+//    private Label text;
+//    private ImageView displayPicture;
+//
+//    public DialogBox(Label l, ImageView iv) {
+//        text = l;
+//        displayPicture = iv;
+//
+//        text.setWrapText(true);
+//        displayPicture.setFitWidth(100.0);
+//        displayPicture.setFitHeight(100.0);
+//
+//        this.setAlignment(Pos.TOP_RIGHT);
+//        this.getChildren().addAll(text, displayPicture);
+//    }
+//
+//    /**
+//     * Flips the dialog box such that the ImageView is on the left and text on the right.
+//     */
+//    private void flip() {
+//        this.setAlignment(Pos.TOP_LEFT);
+//        ObservableList<Node> tmp = FXCollections.observableArrayList(this.getChildren());
+//        FXCollections.reverse(tmp);
+//        this.getChildren().setAll(tmp);
+//    }
+//
+//    public static DialogBox getUserDialog(Label l, ImageView iv) {
+//        return new DialogBox(l, iv);
+//    }
+//
+//    public static DialogBox getDukeDialog(Label l, ImageView iv) {
+//        var db = new DialogBox(l, iv);
+//        db.flip();
+//        return db;
+//    }
+//
+//}

```

Comment:
Tip for commenting large blocks of code! You can save some time by closing your comment block with `/*` and `*/` instead of double slashing every line